DANS-KNAW/easy-schema
# EASY-1486: Store all elements of easy-deposit-ui mockup in the ddm

Part of EASY-1486

#### When applied it will
* Add ISNI and ORCID to creator/contributor
* Let default DDM.xsd use this dcx-dai version
* Also, put ddm:temporal in the default DDM.xsd
* Introduce ddm:description and add descriptionType-attribute

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?
in ddm:creator/contributor these snippets should now be valid (only one per type per contributor):

- `<dcx-dai:DAI>info:eu-repo/dai/nl/0123456789</dcx-dai:DAI>` (already possible)
- `<dcx-dai:ORCID>https://orcid.org/0000-0002-1694-233X</dcx-dai:ORCID>`
- `<dcx-dai:ISNI>http://www.isni.org/000000012281955X</dcx-dai:ISNI>`
- `<dcx-dai:ISNI>ISNI:000000012281955X</dcx-dai:ISNI>`
- `<dcx-dai:ISNI>0000 0001 2281 955X</dcx-dai:ISNI>`

In the ddm:profile and ddm:dcmiMetadata, the following should now be valid:
-`<dc:description>desc</dc:description>`
-`<dcterms:description>desc</dcterms:description>`
-`<ddm:description xml:lang="eng" descriptionType="Abstract">desc</ddm:description>`
-`<ddm:description xml:lang="eng" descriptionType="TechnicalInfo">instructions of use</ddm:description>`
